tests/unit/other.tcl

```diff
@@ -429,6 +429,7 @@ start_server {tags {"other external:skip"}} {
 }
 
 start_cluster 1 0 {tags {"other external:skip cluster slow"}} {
```

Comment:
this test doesn;t run in CI, is it because it's tagged "slow"? is it actually slow?